### ISSUE TITLE
Unable to add linked dependency: `oao add sub-package scoped-dependency`

### ISSUE THREAD

**Issue by ericclemmons** [guigrpa/oao#45]
Suppose I have this:

```
packages/
  example-a/  (example-a)
  scoped-b/  (@scoped/b)
```

When I run:

```shell
$ oao add example-a @scoped/b
```

Yarn attempts to run `yarn install @scoped/b` (which doesn't exist yet).

**Comment by guigrpa** [comment]
Still interested in this, or using lerna instead?

**Comment by ericclemmons** [comment]
Still interested in this.  Not using lerna because oao works better.

In the interim, I'm manually adding `"@scoped/b": "*"` to `package.json` within `example-a` right now.

**Comment by guigrpa** [comment]
Fixed in v0.10.1, but please check in your case.
